.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
   ssh_known_hosts:
     - github.com
 node_js:
-  - 10
```

Comment:
I wonder if we should support 10 as well until EOL for node 10 or should we switch over support to 12 only
@bookernath thoughts ?

Reply:
We should take into account that there are some dependencies (esp. related to hapi) that have last versions already not supporting node 10

Reply:
Decided that we should support node 10 too

Reply:
@MaxGenash lets keep node 10 for now since it is an LTS version. We will drop it next year.

Reply:
Ok.
@junedkazi also just for curiosity tried node 14, but unfortunately, our old version of node-sass doesn’t work with it :(

Reply:
yes since we are on a forked version of node-sass we have not compiled a node 14 binary for it.

Reply:
ok, got it